releases/v0.1.21.yml

```diff
@@ -0,0 +1,137 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: digitalocean-cloud-controller-manager
+  namespace: kube-system
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  template:
+    metadata:
+      labels:
+        app: digitalocean-cloud-controller-manager
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/#marking-pod-as-critical

It seems like this annotation has been deprecated in favor of `priorityClassName`. It may be worth investigating the newer alternative format in the future. I will create a ticket to track

Reply:
good point 👍 

I'm okay with both an issue and a PR that updates the v0.1.21 manifest right away.